### ISSUE TITLE
getting only 30 points, then cutting off.

### ISSUE THREAD

**Issue by drrehak** [plotly/arduino-api#35]
mine was getting cut off after sending 30 data points.  This is quite different then plot.ly displaying only the most recent 30.  which seems to be the variable that we pass in Arduino IDE (graph.maxpoints).  so, I looked in plotly_streaming_cc3000.h
changed 
maxpoints = 30 
to 300.  It seems to be chugging along now past 30. 

Can I remove maxpoints?  and it will stream indefinitely?

**Comment by alexander-daniel** [comment]
No, but you can make it about 20,000. Your browser might experience slowdowns, as the max_points attribute refers to the max points displayed in the browser at any given time.

**Comment by drrehak** [comment]
ok.  so it appears to be very unpredictable.  I can't keep a stream open.  any ideas?

**Comment by alexander-daniel** [comment]
We're currently working on upgrading the stability of our Arduino servers. I'll keep you posted here. About how long does it go for before it fails?

Thanks

**Comment by drrehak** [comment]
Looks like it's usually about an hour.
